perception/iterative_closest_point.py

```diff
@@ -0,0 +1,310 @@
+# Code adapted from Clay Flannigan
+import numpy as np
+
+import meshcat
+import meshcat.geometry as g
+import meshcat.transformations as tf
+
+from sklearn.neighbors import NearestNeighbors
+
+def make_meshcat_color_array(N, r, g, b):
```

Comment:
It would be nice if we could consistently follow drake's style guide: https://drake.mit.edu/code_style_guide.html#python-style . Function names should be CamelCase verb phrases.

Reply:
Identifiers can be changed conveniently by using pycharm's refactor.

Reply:
Thanks for pointing this out! I’ll get to this and the other style comments once I have everything I want in this PR functionality-wise.